recipe/meta.yaml

```diff
@@ -28,17 +28,20 @@ requirements:
     - numpy
     - numpydoc
     - ophyd >=1.5.0
+    - pcdsdevices
```

Comment:
I don't think this is ever imported in typhos

Reply:
Agreed, only for testing (variety IOC)